pkg/controller/.import-restrictions

```diff
@@ -101,13 +101,18 @@
         "github.com/google/gofuzz",
         "github.com/prometheus/client_golang/prometheus",
         "github.com/robfig/cron",
+        "github.com/spf13/afero",
```

Comment:
No -- it has nothing to do with the kubelet change. The reasons is:

For the `ToPlunginHandler` function, we need the type `pluginwatcher.PluginHandler`, therefore we import the package `pluginwatcher` in the `csi` package. A bit more about that dependencies can be found [in the other comment](https://github.com/kubernetes/kubernetes/pull/73919#discussion_r269077222), but in short: those new dependencies are all transitive dependencies, brought in by the `pluginwatcher` package and because the `controller/volume/...` package has a dependency on the `csi` package.